pkg/harvester/dialog/CloneVmDialog.vue

```diff
@@ -91,15 +91,22 @@ export default {
     </template>
 
     <template #body>
-      <Checkbox v-model="cloneData" class="mb-10" label-key="harvester.modal.cloneVM.type" />
-
-      <LabeledInput
-        v-show="cloneData"
-        v-model="name"
-        class="mb-20"
-        :label="t('harvester.modal.cloneVM.name')"
-        required
-      />
+      <template v-if="!actionResource?.longhornV2Volumes.length">
+        <Checkbox v-model="cloneData" class="mb-10" label-key="harvester.modal.cloneVM.type" />
+
+        <LabeledInput
+          v-show="cloneData"
+          v-model="name"
+          class="mb-20"
+          :label="t('harvester.modal.cloneVM.name')"
+          required
+        />
+      </template>
+      <template v-else>
+        <Banner color="warning">
+          <t k="harvester.modal.cloneVM.message.support.longhorn" :raw="true" />
```

Comment:
nit: it's fine to hard code the doc url in en-us.yaml in harvester/dashboard as we will maintain v1.4.x release in this repo.

But when we port the same change to harvester/harvester-ui-extension. The doc version should be varaible. Like https://github.com/harvester/dashboard/blob/b4a6ed9dab3ac2cbbd6d174e725ae5ab00ee9d88/pkg/harvester/components/settings/storage-network.vue#L214